src/app/legacy/containers/StoryPromo/index.jsx

```diff
@@ -196,6 +196,8 @@ const StoryPromoContainer = ({
 
   const useLargeImages = promoType === 'top' || promoType === 'leading';
 
+  const isFirstPromo = index === 0 && useLargeImages;
```

Comment:
I am wondering if we should now rename this so that it makes sense when used with the first promo...?

We would then need to rename all references to useLargeImages, but I don't think there are that many?

```suggestion
  const isTopOrLeadingPromo = promoType === 'top' || promoType === 'leading';

  const isFirstPromo = index === 0 && isTopOrLeadingPromo;
```